oak_remote_attestation/proto/v1/evidence.proto

```diff
@@ -0,0 +1,119 @@
+//
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
```

Comment:
How do the proto definitions in this file relate to the ones in `oak_remote_attestation/proto/v1/messages.proto`? I could not find a high-level description or implementation plan anywhere.

Reply:
They will replace the definitions from `messages.proto` once we have DICE support. The implementation plan is in an internal document for now. Once it's finalized I'll update external documentation.